sphinx/search/non-minified-js/norwegian-stemmer.js

```diff
@@ -1,1771 +1,261 @@
+
+/**@constructor*/
+NorwegianStemmer = function() {
+    var base = new BaseStemmer();
+    /** @const */ var a_0 = [
+        ["a", -1, 1],
+        ["e", -1, 1],
+        ["ede", 1, 1],
+        ["ande", 1, 1],
+        ["ende", 1, 1],
+        ["ane", 1, 1],
+        ["ene", 1, 1],
+        ["hetene", 6, 1],
+        ["erte", 1, 3],
+        ["en", -1, 1],
+        ["heten", 9, 1],
+        ["ar", -1, 1],
+        ["er", -1, 1],
+        ["heter", 12, 1],
+        ["s", -1, 2],
+        ["as", 14, 1],
+        ["es", 14, 1],
+        ["edes", 16, 1],
+        ["endes", 16, 1],
+        ["enes", 16, 1],
+        ["hetenes", 19, 1],
+        ["ens", 14, 1],
+        ["hetens", 21, 1],
+        ["ers", 14, 1],
+        ["ets", 14, 1],
+        ["et", -1, 1],
+        ["het", 25, 1],
+        ["ert", -1, 3],
+        ["ast", -1, 1]
+    ];
+
+    /** @const */ var a_1 = [
+        ["dt", -1, -1],
+        ["vt", -1, -1]
+    ];
+
+    /** @const */ var a_2 = [
+        ["leg", -1, 1],
+        ["eleg", 0, 1],
+        ["ig", -1, 1],
+        ["eig", 2, 1],
+        ["lig", 2, 1],
+        ["elig", 4, 1],
+        ["els", -1, 1],
+        ["lov", -1, 1],
+        ["elov", 7, 1],
+        ["slov", 7, 1],
+        ["hetslov", 9, 1]
+    ];
+
+    /** @const */ var /** Array<int> */ g_v = [17, 65, 16, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 48, 0, 128];
+
+    /** @const */ var /** Array<int> */ g_s_ending = [119, 125, 149, 1];
+
+    var /** number */ I_x = 0;
+    var /** number */ I_p1 = 0;
+
+
+    /** @return {boolean} */
+    function r_mark_regions() {
+        I_p1 = base.limit;
+        var /** number */ v_1 = base.cursor;
+        {
+            var /** number */ c1 = base.cursor + 3;
+            if (c1 > base.limit)
+            {
+                return false;
+            }
+            base.cursor = c1;
+        }
+        I_x = base.cursor;
+        base.cursor = v_1;
+        golab0: while(true)
+        {
+            var /** number */ v_2 = base.cursor;
+            lab1: {
+                if (!(base.in_grouping(g_v, 97, 248)))
+                {
+                    break lab1;
+                }
+                base.cursor = v_2;
+                break golab0;
+            }
+            base.cursor = v_2;
+            if (base.cursor >= base.limit)
+            {
+                return false;
+            }
+            base.cursor++;
+        }
+        golab2: while(true)
+        {
+            lab3: {
+                if (!(base.out_grouping(g_v, 97, 248)))
+                {
+                    break lab3;
+                }
+                break golab2;
+            }
+            if (base.cursor >= base.limit)
+            {
+                return false;
+            }
+            base.cursor++;
+        }
+        I_p1 = base.cursor;
+        lab4: {
+            if (!(I_p1 < I_x))
+            {
+                break lab4;
+            }
+            I_p1 = I_x;
+        }
+        return true;
+    };
```

Comment:
Codacy has a fix for the issue: [Unnecessary semicolon.](https://app.codacy.com/gh/angrycaptain19/sphinx/pullRequest?prid=7480054)
```suggestion
    }
```